nodes/Apify/resources/actors/scrape-single-url/properties.ts

```diff
@@ -0,0 +1,73 @@
+import { INodeProperties } from 'n8n-workflow';
+
+import * as helpers from '../../../helpers';
+
+export const properties: INodeProperties[] = [
+	{
+		displayName: 'URL',
+		name: 'url',
+		description: 'URL to be scraped.',
+		default: 'https://docs.apify.com/academy/web-scraping-for-beginners',
+		type: 'string',
+		routing: {
+			request: {
+				body: {
+					customBody: {
+						startUrls: [{ url: 'https://docs.apify.com/academy/web-scraping-for-beginners' }],
+					},
```

Comment:
Important: the value is hardcoded here
```suggestion
						startUrls: [{ url: '={{ $value }}' }],
					},
```